src/theme/media.js

```diff
@@ -2,6 +2,11 @@ import { css } from 'styled-components'
 import theme from './theme'
 
 export const media = {
+  mobile: (...args) => css`
```

Comment:
This should probably not exist. All styles apply to mobile unless specifically defined for higher screen widths.
```js
styled.blah`
   ${media.mobile`xyz`}
`
```
is functionally the same as
```js
styled.blah`
  xyz
`
```

Reply:
see above, to cater for xs and s mobile sizes